### DISCUSSION TITLE
How can i get node information with MQTT

### DISCUSSION THREAD

**Discussion by cyberumut** [zwave-js/zwave-js-ui#3533] [Q&A]
Im using this method with mqtt but not working 
https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=node-information

<mqtt_prefix>/<?node_location>/<node_name>/nodeinfo

zwave/zwave2mqtt/endnode/nodeinfo  

nothing's coming value .
and whenever I clear the values, when I want to read again, it does not read without setting the values.

**Comment by cyberumut** [comment]
<img width="1058" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/105754498/735e3537-5f51-4605-96d4-dce4649c2739">
but it's visible here

**Comment by robertsLando** [comment]
First the screenshot you sent here shows some node values related to Command class 134 and not node informations. Second I suggest you to use mqtt explorer and see the topics you get there as I think you may be looking at the wrong topic. Topics depends on the mqtt settings and also on node names/location

**Reply by cyberumut** [reply]
when I change the topic name and want to read the firmware etc. I can't read anything, like it doesn't update

**Reply by robertsLando** [reply]
What mqtt client are you using? Can you show logs?

**Reply by cyberumut** [reply]
mosquitto
<img width="659" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/105754498/208bb851-8c61-486f-a350-cd3e5deea465">

**Reply by cyberumut** [reply]
when I reboot, the information comes up

**Reply by robertsLando** [reply]
Please use https://github.com/thomasnordquist/MQTT-Explorer and subscribe to # (default). Explore the toppics tree and everything will be easier

**Reply by cyberumut** [reply]
<img width="605" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/105754498/fe85a0b8-e0d8-4d52-a8fa-8a97b5b68c66">
same problem when change topic name nothing's coming update.

when I restart then the data comes to the new topic name

**Reply by robertsLando** [reply]
that doesn't make sense... that's the name of the node and the location of the node, before them there should be the prefix that you set in mqtt settings... Please send a screenshot of the mqtt settings
